src/metatensor/models/cli/train_model.py

```diff
@@ -336,13 +341,43 @@ def _train_model_hydra(options: DictConfig) -> None:
         device_str=options["device"],
     )
 
-    save_model(model, options["output_path"])
+    save_model(model, f'{options["output_path"][:-3]}.ckpt')
```

Comment:
`[:-3]` looks a bit inelegant for remove the suffix? 

You can use[ Pathlibs stem](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.stem) method for this which maybe makes it a bit clearer.

```suggestion
    save_model(model, Path(options["output_path"]}).stem / ".ckpt")
```

Maybe also a file suffix check could be useful...